vowpalwabbit/core/src/reductions/bfgs.cc

```diff
@@ -1141,7 +1141,7 @@ std::unique_ptr<options_bfgs_v1> get_bfgs_options_instance(
                .default_value(0.001f)
                .help("Termination threshold"))
       .add(make_option("early_terminate", bfgs_opts->early_terminate)
-               .default_value(0)
+               .default_value(3)
```

Comment:
Why change this?

Reply:
The default in parse_args is 3 and before this was using 
`options.get_typed_option<uint64_t>("early_terminate").value();`
which would default to 3, so this keeps behavior the same